sentry-ruby/lib/sentry/transaction_event.rb

```diff
@@ -49,5 +54,33 @@ def to_hash
       data[:measurements] = @measurements
       data
     end
+
+    private
+
+    def populate_profile(transaction)
+      return nil unless transaction.profiler
+
+      profile_hash = transaction.profiler.to_hash
+      return nil unless profile_hash
+
+      profile_hash.merge!(
+        environment: environment,
+        release: release,
```

Comment:
IIRC, information like environment or release are also in `context`. Can't Sentry get them from the associated transaction's context?

Reply:
I see, thanks for the context 👍